scripts/apps/authoring/authoring/directives/SendItem.js

```diff
@@ -345,6 +429,59 @@ export function SendItem($q, api, desks, notify, authoringWorkspace,
             };
 
             /**
+             * Set the User Action.
+             */
+            scope.setUserAction = function(action) {
+                if (scope.currentUserAction === action) {
+                    return;
+                }
+                scope.currentUserAction = action;
+                storage.setItem(USER_ACTION_KEY, action);
+                setDesksAndStages();
+            };
+
+            /**
+             * Send the current item to different desk or stage and then publish the item
+             */
+            function runSendAndPublish() {
+                var deskId = scope.selectedDesk._id;
+                var stageId = scope.selectedStage._id || scope.selectedDesk.incoming_stage;
+                // send releases lock, increment version.
+                return sendAuthoring(deskId, stageId, scope.selectedMacro, true)
+                    .then(function (item) {
+                        // open the item for locking and publish
+                        scope.loading = true;
```

Comment:
Is this the correct place to initialise loading? Maybe it should be on L449, and this should just be:
```js
.then(item => authoring.open(scope.item._id, false)
```
?